### PR TITLE
build: release: v1.27.0-rc1

### PR DESCRIPTION
**Pull request by rjan90**
## Proposed Changes
- Update the build version, and clean up the changelog, highlighting features, etc.

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] If the PR affects users (e.g., new feature, bug fix, system requirements change), update the CHANGELOG.md and add details to the UNRELEASED section.
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,187 @@
 
 ## Improvements
 
+# v1.27.0-rc1 / 2024-04-30
+
+This is the first release candidate of the upcoming optional release of Lotus v1.27.0. This feature release includes numerous improvements and enhancements for node operators, ETH RPC-providers and storage providers. This feature release introduces Curio in Beta release.
```
Comment (jennijuju) [inline CHANGELOG.md:11]: would be great if we can have a section for ETH RPC users (indexers) & list the PR included in this release they would want to check out . not a blocker for RC - but should add for final release.

**Review by jennijuju — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,187 @@
...
+
+## ☢️ Upgrade Warnings ☢️
+
+- This feature release drops the Raft cluster code experiemnt from the codebase. This Raft cluster never graduated beyond an experiment, had poor UX (e.g. no way to manage a running cluster, so it didn't provide High Availability, and pulled in a lot of heavy dependencies. We keep the multi-node RPC feature, it is not perfect, but it is useful.
```
Comment (ZenGround0) [inline CHANGELOG.md:15]: nit: experiment

**Review by ZenGround0 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rvagg) [reply CHANGELOG.md:11]: https://github.com/filecoin-project/lotus/pull/11477 specifically for eth

Not just eth, though

https://github.com/filecoin-project/lotus/pull/11934 is targeting the new actor events RPC APIs

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,187 @@
...
+- chore: switch back to upstream ledger library (#11651) ([filecoin-project/lotus#11651](https://github.com/filecoin-project/lotus/pull/11651))
+- chore: build: update minimum go version to 1.21.7 (#11652) ([filecoin-project/lotus#11652](https://github.com/filecoin-project/lotus/pull/11652))
+- chore: docs: nv-skeleton documentation (#11065) ([filecoin-project/lotus#11065](https://github.com/filecoin-project/lotus/pull/11065))
+- chore: Add v13 support to invariants-checker (#11931) ([filecoin-project/lotus#11931](https://github.com/filecoin-project/lotus/pull/11931))
+
+## Contributors
+
+| Contributor | Commits | Lines ± | Files Changed |
+|-------------|---------|---------|---------------|
+| Rod Vagg | 20 | +55315/-204 | 58 |
```
Comment (rvagg) [inline CHANGELOG.md:154]: whoa, what? I guess this is the .json.gz -> .json change. epic number for a simple change.

**Review by rvagg — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rvagg) [reply CHANGELOG.md:11]: other eth:

 - fix: api: return the correct block gas limit in the EthAPI (#11747) ([filecoin-project/lotus#11747](https://github.com/filecoin-project/lotus/pull/11747))
 - fix: Eth API: accept input data in call arguments under field 'input' (#11505) ([filecoin-project/lotus#11505](https://github.com/filecoin-project/lotus/pull/11505))
- fix: api: Length check the array sent to eth_feeHistory RPC (#11696) ([filecoin-project/lotus#11696](https://github.com/filecoin-project/lotus/pull/11696))
- fix: api: fix EthSubscribe tipsets off by one (#11858) ([filecoin-project/lotus#11858](https://github.com/filecoin-project/lotus/pull/11858))

**Pull request merged**